src/ui/components/Rating/index.js

```diff
@@ -97,7 +97,7 @@ export class RatingBase extends React.Component {
       description = i18n.sprintf(i18n.gettext('Rated %(rating)s out of 5'),
         { rating: i18n.formatNumber(parseFloat(rating).toFixed(1)) });
     } else {
-      description = i18n.gettext('No ratings');
+      description = i18n.gettext('Click the stars to rate this add-on');
```

Comment:
I know this is what's in the issue but I think it was a half throwaway text suggestion. I think even just **"Click to rate this add-on"** would be better, because I think calling them stars isn't that important to screenreaders and sighted users will already see they are stars.